### PR TITLE
Adds `/groups` pages

### PR DESCRIPTION
**Pull request by burkeshartsis**
Current testing ground for groups work.

# Tests
- [x] mobile tested
- [ ] latest update from master

### CODE REVIEW

**Review by rkdavidson — COMMENTED** [review]
Some code comments but otherwise looking good!
Pointed out the empty state for Groups list via Slack so can re-review real quick and approve after.

File: src/groups/GroupContentItemConnected/index.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  const { loading, error, data } = useQuery(GET_GROUP, {
+    variables: { itemId },
+    fetchPolicy: 'cache-and-network',
+  });
+
+  const [handleAttend] = useMutation(ADD_ATTENDANCE);
+
+  if (loading) {
+    return (
+      <div style={{ height: '50vh', width: '100vw', position: 'relative' }}>
```
[the remark this answers is not in the document]
Comment (burkeshartsis) [reply src/groups/GroupContentItemConnected/index.js:25]: This is actually super common all over the current CF codebase so I wasn't sweating it.

File: src/groups/GroupContentItemConnected/GroupContentItem.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+                  {get(videoCall, 'label') || parentVideoCall
+                    ? 'Join Breakout'
+                    : 'Join Meeting'}
+                </a>
+              ) : (
+                <a
+                  // allow to just checkin for in person groups
+                  className="btn btn-primary btn-block mb-3"
+                  href={''}
+                  onClick={() => onClickVideoCall()}
```
[the remark this answers is not in the document]
Comment (burkeshartsis) [reply src/groups/GroupContentItemConnected/GroupContentItem.js:108]: Correct! This handler is only submitting analytics.

**Review by rkdavidson — APPROVED** [review]

**Comment by rkdavidson** [comment]
# ⚠️ NOTE ⚠️ 
## DO NOT MERGE UNTIL MONDAY 9/14! ##

**Pull request merged**